src/menu.scss

```diff
@@ -5,351 +5,306 @@
+      }
     }
+  }
 
-    @include fd-fiori-focus();
+  &__title {
+    @include fd-reset();
+    @include fd-ellipsis();
 
-    @include fd-disabled() {
-      opacity: var(--sapContent_DisabledOpacity);
-      cursor: not-allowed;
-      text-decoration: none;
-      pointer-events: none;
-    }
+    width: 100%;
+    color: var(--sapList_TextColor);
+    font-size: var(--sapFontSize);
+    font-family: $fd-menu-title-font-family;
+  }
 
-    &[aria-disabled='true'] {
-      @include fd-disabled();
-    }
+  &__addon-before,
+  &__addon-after {
+    @include fd-reset();
+    @include fd-flex-center();
 
-    .#{$block}__title {
-      @include fd-reset();
+    color: $fd-menu-icon-color;
+    width: $fd-menu-icon-width;
+    min-width: $fd-menu-icon-width;
+    height: 1.25rem; // to account for item top padding of 12px or 0.75rem so 2.75rem-0.75rem-0.75rem = 1.25rem;
+    font-size: var(--sapFontLargeSize);
+    background-color: transparent;
+  }
 
-      width: 100%;
-      color: $fd-menu-item-color;
+  &__addon-before {
+    margin: $fd-menu-icon-before-margin;
 
-      @include fd-weight("normal");
+    @include fd-rtl() {
+      margin: $fd-menu-icon-after-margin;
+    }
+  }
 
-      font-size: var(--sapFontSize);
-      font-family: var(--sapFontFamily);
-      text-decoration: none;
+  &__addon-after {
+    flex: 0 0 auto;
+    margin: $fd-menu-icon-after-margin;
 
-      @include fd-ellipsis();
+    @include fd-rtl() {
+      margin: $fd-menu-icon-before-margin;
     }
 
-    &.has-child {
-      @include selected-menu();
+    &--submenu {
+      @include fd-icon("navigation-right-arrow", "s");
+
+      font-size: var(--sapFontSmallSize);
 
-      &.is-expanded,
-      &[aria-expanded="true"] {
-        @include fd-selected() {
-          @include selected-menu();
-        }
+      @include fd-rtl() {
+        @include fd-icon("navigation-left-arrow", "s");
       }
     }
   }
 
-  &__addon-before {
+  &__shortcut {
     @include fd-reset();
-    @include fd-flex-center();
+    @include fd-ellipsis();
 
-    // cozy tablet icon size
-    min-width: 2.25rem; // to account for item right padding of 16px or 1rem so 2.25rem-1rem = 1.25rem
-    width: 2.25rem; // to account for item right padding of 16px or 1rem so 2.25rem-1rem = 1.25rem
-    height: 1.25rem; // to account for item top padding of 12px or 0.75rem so 2.75rem-0.75rem-0.75rem = 1.25rem
-    margin: 0 0 0 -0.75rem;
+    display: flex;
+    flex-shrink: 0;
+    font-weight: normal;
+    font-family: $fd-menu-title-font-family;
+    color: var(--sapContent_NonInteractiveIconColor);
+    font-size: var(--sapFontSize);
+    padding-left: 1rem;
+  }
 
-    @include fd-rtl() {
-      margin: 0 -1rem 0 0; // since before and after icons switch places, their margins also switch
-    }
+  &--full-width {
+    max-width: 100%;
+  }
+
+  &--compact {
+    .#{$block}__link {
+      padding: 0.375rem 0.75rem; // line-height=20px so 2rem or 32px-20px=12px/2= 6px or 0.375rem each;
```

Comment:
Maybe this should also be in variables

Reply:
Also could you remove this comment? I don't think if it describes anything there.

Reply:
This value is used only in this one place.